src/package/nuspec/Microsoft.CodeCoverage.nuspec

```diff
@@ -73,6 +73,10 @@
     <file src="Microsoft.CodeCoverage\InstrumentationEngine\alpine\x64\libInstrumentationEngine.so"	target="build\netstandard1.0\InstrumentationEngine\alpine\x64\libInstrumentationEngine.so" />
     <file src="Microsoft.CodeCoverage\InstrumentationEngine\alpine\x64\VanguardInstrumentationProfiler_x64.config"	target="build\netstandard1.0\InstrumentationEngine\alpine\x64\VanguardInstrumentationProfiler_x64.config" />
 
+    <file src="Microsoft.CodeCoverage\InstrumentationEngine\macos\x64\libCoverageInstrumentationMethod.dylib"	target="build\netstandard1.0\InstrumentationEngine\macos\x64\libCoverageInstrumentationMethod.dylib" />
```

Comment:
I am assuming these new files are signed?

Reply:
No signing for linux/macOS binaries :)